src/keynames.hpp

```diff
@@ -0,0 +1,1389 @@
+const int VoidSymbol=0xFFFFFF; //Does nothing
```

Comment:
Remove empty comments (or I can do this with couple of key presses in vim)

Reply:
I think some of them are useful? Is it really needed?
TODO: add all different numbers that are not list (arabic etc.), add latin alphabet (this is for me)

Reply:
oh, EMPTY comments. Removed